tests/test_dsl.py

```diff
@@ -255,6 +256,30 @@ def create(self) -> tosca.artifacts.Root:
     }
 
 
+from unittest.mock import MagicMock
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        (dict(file="foo.yaml"), "from .foo import *"),
+        (dict(file="../foo.yaml"), "from ..foo import *"),
+        (dict(file="../../foo.yaml"), "from ...foo import *"),
+        (dict(repository="unfurl", file="foo.yaml"), "from unfurl.foo import *"),
+        (dict(repository="repo", file="foo.yaml", namespace_prefix="tosca.ns", uri_prefix="tosca.ns"), "from repo import foo as tosca.ns"),
```

Comment:
this should be `tosca_repositories.repo` see this code in `find_repository` 
```        if name in self.repository_paths:
            return "tosca_repositories." + name, self.repository_paths[name]
```
i'm surprised the unit test didn't raise an error because of these lines in that function 
```
        tpl = self.template.tpl["repositories"][tosca_name or name]
        url = normalize_path(tpl["url"])
```

Reply:
ah -- that only happens if the given repo is a URL; i'll add a test case for that

Reply:
The repo is the name of repository not the url itself... I think the problem is the mock does have a repositories dictionary with the repo as a key